src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -64,8 +64,22 @@ public static ColumnConcatenatingEstimator Concatenate(this TransformsCatalog ca
         /// DropColumns is used to select a list of columns that user wants to drop from a given input. Any column not specified will
         /// be maintained in the output schema.
         /// </summary>
+        /// <remarks>
+        /// DropColumns is commonly used to remove unwanted columns from the schema if the dataset is going to be serialized or
+        /// written out to a file. It is not actually necessary to drop unused columns before training or
+        /// performing transforms, as IDataView's lazy evaluation won't actually materialize those columns.
```

Comment:
Could we have things like `<see cref="IDataView">` for types when possible (an example is [here](https://github.com/dotnet/machinelearning/blob/e5bdd10d78400a51fde9e30846c37837ed40eb01/src/Microsoft.ML.FastTree/RegressionTree.cs#L50))?

Reply:
Why don't we make a `using`?